FAQ_ptBR.md

```diff
@@ -0,0 +1,31 @@
+# Memo - Perguntas Frequentes
+
+- [Memo - Perguntas Frequentes](#memo---perguntas-frequentes)
+  - [O que é uma Coleção?](#o-que-é-uma-coleção)
+  - [O que é um Memo?](#o-que-é-um-memo)
+  - [O que é o nível de fixação?](#o-que-é-o-nível-de-fixação)
+
+## O que é uma Coleção?
+
+Uma coleção se trata de um grupo de [Memos](#o-que-é-um-memo).
+
+Fazendo uma analogia com [flashcards](https://en.wikipedia.org/wiki/Flashcard), ela é o que consideramos um Deck.
+
+## O que é um Memo?
+
+Um memo é um "desafio" individual de uma [Coleção](#o-que-é-uma-coleção).
+
+Fazendo uma analogia com [flashcards](https://en.wikipedia.org/wiki/Flashcard), é o que consideramos como uma Carta,
+esta na qual pertence à um Deck.
```

Comment:
`esta na qual`? It doesn't seem correct to be. Wouldn't it be just `a qual...`?

Reply:
I don't think so. In this case, I specifically refer to "esta na qual" as the "Carta" reference.